src/site/content/en/blog/serial/index.md

```diff
@@ -4,15 +4,13 @@ subhead: The Serial API allows websites to communicate with serial devices.
 authors:
   - beaufortfrancois
 date: 2020-08-12
-updated: 2020-12-02
+updated: 2021-01-06
```

Comment:
I believe these dates affect RSS feeds. Please update these before going live.

Reply:
I will.

Reply:
I've updated it.